src/app/layout/status-bar/status-bar.component.scss

```diff
@@ -9,6 +9,14 @@
     height: 2em;
     line-height: $layout-stripe-width - 6;
 
+    .panel-toggle {
```

Comment:
Not needed (except for font-size) if put the button.panel-toggle inside a div.btn-group.